dashboard/app/views/home/_homepage.html.haml

```diff
@@ -23,22 +23,10 @@
 #homepage-container
 
 #workshop-links.container.main
-  -if can? :read, Pd::Application::ApplicationBase
```

Comment:
These links should to more than Workshop Admins and until the work to add the Links for RPs, Facilitators and Workshop organizers is done I don't think we can remove

Reply:
Oh I see what you did. You are allows the same people to see the gray box. I don't think we want RPs, Facilitators and Workshop Organizers to see that gray box. Just internal Workshop Admins

Reply:
oh woops -- I misunderstood! I'll move this PR back to draft and update

Reply:
@dmcavoy circling back to this. In [the spec](https://docs.google.com/document/d/1Ee02kw_ei2x6wOS06K768dC-kUhGDlydjoE6BZipbPE/edit#bookmark=kix.o5cbqsv9l6ei), these are the links listed to move to the gray box. I see that the application dashboard, workshop dashboard, etc are moving to other places as well. Should those links not be in the gray box at all?

Reply:
No issues with the links. I think you have the right list of links. What I was commenting on was the permissions. As a whole this gray box should only show for Workshop Admins (internal Code.org staff who manage workshop and application stuff). Workshop Organizers, Regional Partners, Facilitators, etc will all have access to these links right on the page. Does that help?

Reply:
I think so!